### PR TITLE
Fix MISRA violations 8.9

### PR DESCRIPTION
**Pull request by alfred2g**
Fix MISRA violations 8.9

Description
-----------
Fix MISRA violations 8.9

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by htibosch — APPROVED** [review]
The changes proposed in this PR look harmless to me, but I do miss a proper explanation why you want to introduce these changes. Please explain a bit more,
Thanks, Hein

**Comment by alfred2g** [comment]
> The changes proposed in this PR look harmless to me, but I do miss a proper explanation why you want to introduce these changes. Please explain a bit more, Thanks, Hein

Thanks for taking a look!
We are fixing all misra violations, and Rule 8.9 likes all globals/statics  that are used in one function to be defined in that function.
Alfred

File: source/FreeRTOS_ARP.c [hunk 1]
```diff
@@ -141,6 +129,17 @@ eFrameProcessingResult_t eARPProcessPacket( ARPPacket_t * const pxARPFrame )
...
     void * pvCopyDest;
 
+    /* Next defensive request must not be sent for arpIP_CLASH_RESET_TIMEOUT_MS
+     * period. */
+    static TickType_t uxARPClashTimeoutPeriod = pdMS_TO_TICKS( arpIP_CLASH_RESET_TIMEOUT_MS );
+
+    /* This local variable is used to keep track of number of ARP requests sent and
+     * also to limit the requests to arpIP_CLASH_MAX_RETRIES per arpIP_CLASH_RESET_TIMEOUT_MS
+     * period. */
+    static UBaseType_t uxARPClashCounter = 0U;
```
Comment (paulbartell) [inline source/FreeRTOS_ARP.c:139]: Does this work for multiple interfaces?

Comment (alfred2g) [reply source/FreeRTOS_ARP.c:139]: it shouldn't work without synchronization from the interface

**Pull request merged**